values.yaml

```diff
@@ -814,6 +814,16 @@ redis:
     affinity: {}
     ## The priority class to run the pod as
     priorityClassName:
+    # # jobserviceDatabaseIndex defaults to "1"
+    # # registryDatabaseIndex defaults to "2"
+    # # trivyAdapterIndex defaults to "5"
+    # # harborDatabaseIndex defaults to "0", but it can be configured to "6"
+    # # cacheLayerDatabaseIndex defaults to "0", but it can be configured to "7"
+    jobserviceDatabaseIndex: "1"
+    registryDatabaseIndex: "2"
+    trivyAdapterIndex: "5"
+    # harborDatabaseIndex: "6"
```

Comment:
is it better to add a comment clarify it is optional for  `harborDatabaseIndex` and `cacheLayerDatabaseIndex` ?

Reply:
add the comment in line 820 & 821